### DISCUSSION TITLE
Acceder a datos de un widget específico

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#35] [Announcements]
# Acceder a datos de un widget específico

Aprende a acceder a los datos de un widget en específico desde cualquier parte del código de una plantilla de Blogger.

https://www.zkreations.com/2024/04/acceder-datos-de-un-widget-especifico.html

**Comment by adicore** [comment]
Can this method be used to get data from b:widget-setting?

**Reply by danieIabel** [reply]
Good morning, my friend,

I apologize for the delay in my response; I didn't notice your comment. Regarding what you mentioned, it is currently not possible to access data from those tags from another widget. You can only access the configuration that is available in the global data tags.
